apps/front/pages/trip.vue

```diff
@@ -0,0 +1,326 @@
+<template>
+  <div v-if="!loading" class="relative w-screen h-screen">
+    <TripMap class="absolute z-0" :routes="routes" />
+    <div
+      class="absolute bottom-0 lg:top-0 left-0 right-0 lg:w-1/3 p-2 lg:p-12 lg:m-6 z-10 flex flex-col"
+      :class="{ 'rounded-l': drawerOpen, rounded: !drawerOpen }"
```

Comment:
rounded-l does not exists
```suggestion
      :class="{ 'rounded-lg': drawerOpen, rounded: !drawerOpen }"
```